types/Keyguard.d.ts

```diff
@@ -99,7 +197,11 @@ type ConstructSwap<T extends KeyguardRequest.SignSwapRequestCommon> = Transform<
             locktime?: number;
             refundKeyPath: string,
             refundAddress: string,
-        } | {
+        } | (
+            { type: 'USDC' } & KeyguardRequest.PolygonTransactionInfo & {
```

Comment:
```suggestion
            { type: 'USDC' } & Omit<KeyguardRequest.PolygonTransactionInfo, 'amount'> & {
```
I believe the amount can be omitted.

Can also use the `Transform` utility to make this a bit nicer.